res/controllers/Traktor-Kontrol-S3-hid-scripts.js

```diff
@@ -680,10 +681,16 @@ TraktorS3.Deck.prototype.jogHandler = function(field) {
         if (this.wheelTouchInertiaTimer !== 0) {
             return;
         }
-        let playPosition = engine.getValue(this.activeChannel, "playposition");
-        playPosition += deltas[0] / 2048.0;
-        playPosition = Math.max(Math.min(playPosition, 1.0), 0.0);
-        engine.setValue(this.activeChannel, "playposition", playPosition);
+        // If we spam seeks on every single update, it can cause problems with HQ
+        // Rubberband.
+        now = Date.now();
+        if (now - this.seekRateLimitLastTick > 10) {
+            this.seekRateLimitLastTick = now;
+            let playPosition = engine.getValue(this.activeChannel, "playposition");
+            playPosition += deltas[0] / 256.0;
```

Comment:
this is for quick seeking through a track, and we have not to my knowledge preserved sticker position on these large seeks.  I chose these magic numbers based on what was a comfortable speed for seeking through tracks.  I don't think there needs to be a specific calculation involved.

deltas[1] is the timestamp value which is not even going to be used after we merge the other big S3 change. This is an intermediate fix to prevent a party-stopper issue with the current configuration -- loud static out of the main speakers when seeking a track even if the track is not being output to master.

Reply:
I can tell it to divide by deltas[1] but I'm not sure it will make much difference